src/meta/src/controller/utils.rs

```diff
@@ -513,7 +524,7 @@ where
         .await?;
     if count > 0 {
         assert_eq!(count, 1);
-        return Err(MetaError::catalog_duplicated("user", name));
+        return Err(MetaError::catalog_duplicated("user", name, false));
```

Comment:
Personally I would suggest making it new variant of error for better readability.